src/components/buttons/Button.jsx

```diff
@@ -0,0 +1,159 @@
+// @flow
+
+import * as React from 'react';
+import type {Node} from 'react';
```

Comment:
if you're using `React.Node` you don't need this import

Reply:
I have changed it to `import React from 'react';` as it was used this way in other places. hope that's fine.

Reply:
unfortunately if we use `React.Node`, React should be imported as `import * as React from 'react';`, otherwise JSX autocompletion in VSCode won't work :(

Reply:
we checked it and in the current case, it works with both styles of imports, the problem I mentioned was about something different I believe, no actions needed about it.